lib/menu.rb

```diff
@@ -0,0 +1,22 @@
+class Menu
+  attr_reader :menu_items
+
+  FOOD = { pizza: 2.99, burger: 3.50, burrito: 4.50, 
+  lobster: 50.00 }
+
+  def initialize(menu_items = FOOD)
+    @menu_items = menu_items
+  end
+
+  def print
```

Comment:
Just checking that you realise the print and formatted_menu methods are both inaccessible to the user (as menu is injected into order class); so no way to view what's on offer to place an order without separately creating a new menu class. 

Consider having a method in Order that delegates responsibility to Menu? 
```suggestion
(in order.rb)
  def print
    menu.print
  end
  
```